README.md

```diff
@@ -46,15 +51,25 @@ else interest_rate = 10%
 
 **1st example:**
 
-You made your first deposit `1000 tokens` and then made an instant withdrawal in or near this block. In this case, your earned interest will be 0 or very small and you will pay a commission about `30 tokens` and get about `970 tokens` back.
+User deposits `1000 tokens` then makes an instant withdrawal in or near this block. In this case, earned interest will be close to 0 with a fee of about `30 tokens`. User will receive `970 tokens` back.
 
 **2nd example:**
 
-You made your first deposit `1000 tokens`. Then you decided to make one more deposit `1000 tokens` after `2 weeks`. As it is the 1st interval the contract will pay you `5%` annual interest `(1000 * 0.05) / 365 * 14 = 1.92` and it will update your deposit date to the current date. Now you have `2001.92 tokens`, and `3 months` later you decided to withdraw all. `3 months >= (1 month + 2 months)` so this is the 3rd interval. In this case, the contract will pay you `7%` annual interest `(2001.92 * 0.07) / 365 * 90 = 34.55` and you will receive `2036.47 tokens`.
+User deposits `1000 tokens`. After `2 weeks`, the user makes a 2nd deposit of `1000 tokens`. This is within the 1st interval, so the contract will pay  `5%` annual interest `(1000 * 0.05) / 365 * 14 = 1.92` and will update the deposit date to the current date. The new balance will be `2001.92 tokens`.
+
+`3 months` later the user makes a timed withdawal for the total amount. `3 months >= (1 month + 2 months)` which corresponds to the 3rd interval. In this case, the contract will pay `7%` annual interest `(2001.92 * 0.07) / 365 * 90 = 34.55` and the userwill receive `2036.47 tokens`.
 
 **3rd example**
 
-You made your first deposit `1000 tokens`. Then you decided to withdraw a half after `6 months`. As it is the 4th interval the contract will pay you `8%` annual interest `(1000 * 0.08) / 365 * 180 = 3.95` and it will update your deposit date to the current date. Now you have `503.95 tokens`, and `1 year` later you decided to withdraw all. This staking period is greater than the sum of staking intervals so the contract will pay you `10%` annual interest `(503.95 * 0.1) = 50.4` and you will receive `554.35 tokens`.
+User deposits `1000 tokens`. They make a timed withdraw for half after `6 months`. This is the 4th interval corresponding to `8%` annual interest `(1000 * 0.08) / 365 * 180 = 39.45` . User receives `500` STAKE, the remaining half + interest remains in the contract and the deposit date is updated to the current date. 
```

Comment:
```suggestion
User deposits `1000 tokens`. They make a timed withdraw for half after `6 months`. This is the 4th interval corresponding to `8%` annual interest `(1000 * 0.08) / 365 * 180 = 39.45` . User receives `500 tokens`, the remaining half + interest remains in the contract and the deposit date is updated to the current date. 
```
Since we have no mention of STAKE elsewhere in the example